docs/src/examples/redis-clustering-aware.md

```diff
@@ -0,0 +1,89 @@
+# Redis Clustering with cluster aware client
+
+The following guide shows you how to configure Shotover to support proxying Redis cluster *aware* clients to [Redis cluster](https://redis.io/topics/cluster-spec).
+
+## Overview
+
+In this example, we will be connecting to a Redis cluster that has the following topology:
+* `172.16.1.2:6379`
+* `172.16.1.3:6379`
+* `172.16.1.4:6379`
+* `172.16.1.5:6379`
+* `172.16.1.6:6379`
+* `172.16.1.7:6379`
+
+Shotover will be deployed as a sidecar to each node in the Redis cluster, listening on `6380`. Use the following [docker-compose.yml](https://github.com/shotover/shotover-proxy/blob/main/shotover-proxy/examples/redis-cluster-ports-rewrite/docker-compose.yml) to run the Redis cluster and Shotover sidecars.
+
+```console
+curl -L https://raw.githubusercontent.com/shotover/shotover-proxy/main/docs/topology/docker-compose.yml --output docker-compose.yml
```

Comment:
We should show a snippet from the docker-compose.yml that shows how to setup a shotover instance and explain what the different bits mean.